configure.ac

```diff
@@ -7460,6 +7460,15 @@ for builtin_hash in $with_builtin_hashlib_hashes; do
 done
 IFS=$as_save_IFS
 
+# builtin HACL* HMAC module
+AC_MSG_CHECKING([for --with-builtin-hashlib-hmac=yes,no])
```

Comment:
Is there a good reason to have a configure flag for this?  I don't expect anyone to turn it off. If so, document why it's even an option in a comment in here, or in the flags own help text.

Reply:
I wasn't really sure about this. It's possible to disable the `sha3` module, which is also built-in with HACL* now, so I wondered whether it could make sense to disable the HACL* implementation of HMAC for people who wanted to rely on OpenSSL (for whatever reason).

But I think we can definitely remove this option. WDYT?